agents/base.py

```diff
@@ -74,4 +75,24 @@ def get_params(self):
         for key in self._get_param_names():
             params[key] = getattr(self, key)
 
-        return params
\ No newline at end of file
+        return params
+
+    def next_move(self):
+        """
+        Get the next move to be made by the agent.
+
+        Parameters
+        ----------
+        None 
+
+        Returns
+        -------
+        next_move:  integer
+                    Next maneuver to be played by the agent encoded as one of 
+                    the BaseAgent attributes: move_up, move_left, etc.
```

Comment:
This is no longer an integer. We're importing the Selenium WebDriver common keys and using them directly instead of having encoded maneuvers which need to later be translated into the keys.